File: app/src/main/java/com/rkbapps/neetflix/models/person/movie/AsACastModel.java
package com.rkbapps.neetflix.models.person.movie;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class AsACastModel {
    @SerializedName("adult")
    @Expose
    private Boolean adult;
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("poster_path")
    @Expose
    private String posterPath;
    @SerializedName("release_date")
    @Expose
    private String releaseDate;
    @SerializedName("title")
    @Expose
    private String title;
    @SerializedName("vote_average")
    @Expose
    private Double voteAverage;
    @SerializedName("character")
    @Expose
    private String character;
    @SerializedName("order")
    @Expose
    private Integer order;

    public Boolean getAdult() {
        return adult;
    }

    public Integer getId() {
        return id;
    }

    public String getPosterPath() {
        return posterPath;
    }

    public String getReleaseDate() {
        return releaseDate;
    }

    public String getTitle() {
        return title;
    }

    public Double getVoteAverage() {
        return voteAverage;
    }

    public String getCharacter() {
        return character;
    }

    public Integer getOrder() {
        return order;
    }
}

File: app/src/main/java/com/rkbapps/neetflix/models/person/movie/AsACastModel.kt
package com.rkbapps.neetflix.models.person.movie

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class AsACastModel(

    @SerializedName("adult")
    @Expose
    val adult: Boolean,

    @SerializedName("id")
    @Expose
    val id: Int,

    @SerializedName("poster_path")
    @Expose
    val posterPath: String,

    @SerializedName("release_date")
    @Expose
    val releaseDate: String,

    @SerializedName("title")
    @Expose
    val title: String,

    @SerializedName("vote_average")
    @Expose
    val voteAverage: Double,

    @SerializedName("character")
    @Expose
    val character: String,

    @SerializedName("order")
    @Expose
    val order: Int,

    )